pages/governance/proposal/[proposalId].governance.tsx

```diff
@@ -419,7 +420,7 @@ export default function ProposalPage({
                   >
                     <Box sx={{ textAlign: 'right' }}>
                       <Typography>
-                        ~ {dayjs.unix(proposal.creationTimestamp).format('DD MMM YYYY, hh:mm a')}
+                        {dayjs.unix(proposal.creationTimestamp).format('D MMM YYYY, HH:mm UTC Z')}
```

Comment:
To practice DRY code, this statement is similarly used 5 times, and even with the same formatting.  The only thing that differs is what the timestamp is.  Therefore, it makes sense to abstract it out into a reusable function that can take in the timestamp and output a formatted display of it.  This will also reduce the amount of human error that can occur when it comes to these formatting syntax, as well as offering the improvement of being able to maintain the format in one spot and have it updated in many.

You can rewrite this as a single-line function as such in Typescript:

```js
const formatTime = (timestamp: number): string => dayjs.unix(timestamp).format('D MMM YYYY, HH:mm UTC Z')
```

And use it in the five places as such:

```jsx
<Typography>{formatTime(proposal.creationTimestamp)}</Typography>

<Typography>{formatTime(proposal.startTimestamp)}</Typography>

//...etc
```

Furthermore, on this concept of abstraction, if this time format is used in other areas of the application outside of this file, then it makes more sense to pull it out to a separate file that can be imported into many.  The place in the filesystem to put this would be `/src/utils/formatTime.ts` and `export default formatTime`.